src/victory-util/helpers.js

```diff
@@ -140,6 +140,22 @@ export default {
       {};
   },
 
+  modifyProps(props, fallbackProps) {
+    const themeCheck = props.theme && props.theme.props;
```

Comment:
This method should be agnostic of the specific props it need to merge. It should not directly reference `width` or `height` at all.